hbase_sql/src/main/java/com/splicemachine/olap/CompactionJob.java

```diff
@@ -165,15 +166,15 @@ private boolean compactionRunning(List<Integer> jobIds) {
             return false;
```

Comment:
This function checks whether the compaction actually started (or completed) execution, with this change it returns true whenever it is scheduled, but before its tasks start running. 

To use the new API we'd need to call getStageInfo() which has the same activeTasks and completedTasks information we were checking with the old API. You can get the stageId from the jobInfo object (there's only one stage for compactions) and then pass it to statusTracker.getStageInfo()

Reply:
Ok I misunderstood. Thanks for the clarification. Will make necessary changes.